### DISCUSSION TITLE
[Question][Hw4] About the interval filter example given on the description

### DISCUSSION THREAD

**Discussion by yungheelee** [prosyslab-classroom/cs524-program-analysis#338] [Q&A] [answered]
Name: Yunghee Lee

Hello, I have a minor question on the example given on the description of HW4.

> ### 3. Implementing Numerical Domains and Memory Domain
> ...
> `filter < [1, 10] [1, 10]` will return `[1, 10]` because no refinement is possible.

Why is this the case? I think that the concretized numeric value of the right interval `[1, 10]` is at most 10, so that the left interval can be soundly refined into `[1, 9]`.

**Comment by yeonhee-ryou** [comment, marked as answer]
You're right. Sorry for the confusion.

**Comment by yungheelee** [comment]
Thank you for the answer.
